modules/top/routes/reglas.ts

```diff
@@ -6,13 +6,25 @@ import * as reglasCtrl from '../controller/reglas';
 
 const router = express.Router();
 
-router.post('/reglas', (req, res, next) => {
-    let ArrReglas = req.body.reglas;
-    let data = reglasCtrl.guardarReglas(ArrReglas);
-    Promise.all(data.lista).then(resultado => {
-        return (data.resultados);
-    }).catch(error => { return (error); });
-    res.json(data.resultados);
+router.post('/reglas', async (req, res, next) => {
+    const ArrReglas = req.body.reglas;
+    try {
+        if (ArrReglas.length > 0) {
+            const params = {
+                'destino.organizacion.id': ArrReglas[0].destino.organizacion.id,
+                'destino.prestacion.conceptId': ArrReglas[0].destino.prestacion.conceptId,
+            };
+            await reglas.deleteMany(params);
+        }
+        ArrReglas.map(async (regla) => {
```

Comment:
Para grabar en cocunrrencia. Cómo esta ahora no esta esperando todos los save.
```suggestion
       let acciones =  ArrReglas.map(async (regla) => {
            let unaRegla = new reglas(regla);
            Auth.audit(unaRegla, req);
            return unaRegla.save();
        });
        await Promise.all(acciones);
        res.json(ArrReglas);
```

Reply:
Se realiza modificación